common/reviews/api/rush-lib.api.ts

```diff
@@ -384,6 +385,10 @@ module RushConstants {
 
 }
 
+// WARNING: Because this definition is explicitly marked as @internal, an underscore prefix ("_") should be added to its name
+// @internal
+export function start(launcherVersion: string, isManaged: boolean): void;
```

Comment:
>start [](start = 16, length = 5)

Friends don't let friends export global functions.

```
class Rush {
  public static launch(launcherVersion: string, isManaged: boolean): void;
}
```

Reply:
Fine.

---
In reply to: [147528626](https://github.com/Microsoft/web-build-tools/pull/417#discussion_r147528626) [](ancestors = 147528626)